monster.py

```diff
@@ -1,6 +1,68 @@
-from thing import Thing
+import random
 
+from room import Room
+from thing import Thing
+from player import Player
 
 class Monster(Thing):
+    def __init__(self, room: Room, x=0, y=0, player: Player = None):
+        super().__init__(room, x, y)
+
+        self.player = player
+        self.speed = 1
+        
+    def _step(self):
+        nx, ny = 0, 0
```

Comment:
keď sa príšera nehýbe, mala by zostať stáť, teda to chcei iný default
chce tu byť
```python
nx, ny = self.x, self.y
```